heroku-18-build/setup.sh

```diff
@@ -26,6 +26,7 @@ apt-get install -y --no-install-recommends \
     libaudit-dev \
     libbsd-dev \
     libbz2-dev \
+    libc6-dev \
```

Comment:
The `*-build` image variants use the non-build images as their base. As such only build-specific packages need to be listed in `*-build/setup.sh`, since the others will already be installed.
```suggestion
```